Signature format2

julius opened the pull request:
New function sign2() takes the URL-Path instead of controller+action+GET-Params-in-Payload.

julius committed:
Fix existing Tests

julius committed:
Signature Format v2

ghost commented:
### CodeSee Review Map:
[![Review these changes using an interactive CodeSee Map](https://s3.us-east-2.amazonaws.com/maps.codesee.io/images/github/AdmiralCloud/ac-signature/7/bb66ce44/b0de0c6a39c83fae675e3b2e2659e46abef4cdc5.svg)](https://app.codesee.io/r/reviews?pr=7&src=https%3A%2F%2Fgithub.com%2FAdmiralCloud%2Fac-signature)

Review [in an interactive map](https://app.codesee.io/r/reviews?pr=7&src=https%3A%2F%2Fgithub.com%2FAdmiralCloud%2Fac-signature)

View [more CodeSee Maps](https://app.codesee.io/r/maps?src=https%3A%2F%2Fgithub.com%2FAdmiralCloud%2Fac-signature)

#### Legend
![CodeSee Map Legend](https://app.codesee.io/Legend_v2.png)

markBerlin committed:
[signature-format2] package(App): Updated packages | MP

Updated packages
Related issues:
- undefined/browse/signature-format2

markBerlin committed:
[signature-format2] chore(App): Updated lint fix for multiple tests | MP

Updated lint fix for multiple tests
Related issues:
- undefined/browse/signature-format2

markBerlin committed:
[signature-format2] feat(App): Version 2 works with path alone | MP

Instead of controller/action, the new version 2 only requires the path.
Related issues:
- undefined/browse/signature-format2

markBerlin committed:
Merge branch 'master' into signature-format2

* master:
  Create node.js.yml

markBerlin committed:
[signature-format2] chore(App): Version only wotks with Node 12 or above | MP

 Version only wotks with Node 12 or above
Related issues:
- undefined/browse/signature-format2

MrGutis commented on Makefile:
```diff
@@ -2,7 +2,7 @@ MOCHA_OPTS= --slow 0 -A
+	./node_modules/.bin/eslint --fix index.js test/*.js
 
 lint-check:
 	./node_modules/.bin/eslint index.js test/test.js
```
@markBerlin this should be the same as for lint-fix - test/*.js

MrGutis reviewed (approved):
approving but lint-check tries to check non-existent file. See my comments
